### PR TITLE
Refactor unused function args in presto-main

### PR DESCRIPTION
**Pull request by brandboat**
#4188
This PR is a preparation before enable UnusedVariable error prone check.

### CODE REVIEW

File: presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java [hunk 1]
```diff
@@ -486,7 +486,7 @@ public void accept(Throwable throwable)
         }
     }
 
-    private static Type createMapType(Type keyType, Type valueType)
```
Comment (findepi) [inline presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java:489]: instead of removing, use the `valueType` below:
```
return METADATA.getType(mapType(keyType.getTypeSignature(), valueType.getTypeSignature()));
```

this should be as a separate commit

**Review by findepi — APPROVED** [review]
Thanks!

File: presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java (hunk 1 above)
Comment (brandboat) [reply presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java:489]: Oops, I will fix this.

**Comment by brandboat** [comment]
Is there anything need to be revised ?

**Comment by findepi** [comment]
Merged, thanks!
